drivers/clock_control/nrf5_power_clock.c

```diff
@@ -12,10 +12,16 @@
 #include <clock_control.h>
 #include <misc/__assert.h>
 #include <arch/arm/cortex_m/cmsis.h>
+#include <nrf_power.h>
```

Comment:
Have a separate commit for  nrf_power_clock changes.

Reply:
Okay, will do.